index.js

```diff
@@ -26,27 +27,20 @@ function waitFor(ms) {
 
 var app = module.exports = express();
 
-function logArgs() {
-    var args = arguments;
-    process.nextTick(function() {
-        console.log.apply(console, args);
-    });
-}
-
 function funkLogMsg(num, msg) {
```

Comment:
Just wondering if this was a typo of `func`?

Reply:
Honestly not sure; there's a `funk.js` in `lib/`, I think you're generally in the right ballpark...